src/SIS.WebServer/Server.cs

```diff
@@ -31,20 +31,21 @@ public void Run()
             this.listener.Start();
             this.isRunning = true;
 
-            Console.WriteLine($"Server started at http://{LocalhostIpAddress}:{port}");
+            Console.WriteLine($"Server started at http//{LocalhostIpAddress}:{this.port}");
+            while (isRunning)
+            {
+                Console.WriteLine("Waiting for client...");
+
+                var client = listener.AcceptSocketAsync().Result;
```

Comment:
Always use `.GetAwaiter().GetResult()` instead of just `.Result` when making asynchronous call synchronous.